catalog/core/catalog-core-api-impl/pom.xml

```diff
@@ -46,6 +46,16 @@
             <groupId>ddf.platform.util</groupId>
             <artifactId>platform-util</artifactId>
         </dependency>
+        <dependency>
+            <groupId>ddf.platform.util</groupId>
+            <artifactId>platform-util</artifactId>
+            <version>2.9.0-SNAPSHOT</version>
+        </dependency>
+        <dependency>
```

Comment:
Looks like you added this dependency twice. And it should also use ${project.version}